src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -22,6 +25,7 @@ function ProfilePage() {
   useEffect(() => {
     const fetchData = async () => {
       try {
+        // Fetch user data
         const data = await getUserData(userID)
         if (data !== undefined && data !== null) {
           console.log('Fetched user data:', data)
```

Comment:
remove this please. if you need a console log do it on your own branch

Reply:
🫠